db/version_set.cc

```diff
@@ -1310,7 +1310,8 @@ void VersionStorageInfo::ComputeCompactionScore(
       } else {
         score = static_cast<double>(num_sorted_runs) /
                 mutable_cf_options.level0_file_num_compaction_trigger;
-        if (compaction_style_ == kCompactionStyleLevel && num_levels() > 1) {
+        if (score >= 1.0 && compaction_style_ == kCompactionStyleLevel &&
+            num_levels() > 1) {
```

Comment:
Yes, that seems like a good idea so we don't have to change base-level sizing in the current release! Will do it in the morning.